android/assets/jsons/translations/Dutch.properties

```diff
@@ -815,8 +785,7 @@ Avoid [terrain] = Vermijd [terrain]
 The Mayan Long Count = 
  # Requires translation!
 Your scientists and theologians have devised a systematic approach to measuring long time spans - the Long Count. During the festivities whenever the current b'ak'tun ends, a Great Person will join you. = 
- # Requires translation!
-While the rest of the world calls the current year [year], in the Maya Calendar that is: = 
+While the rest of the world calls the current year [year], in the Maya Calendar that is: = Terwijl de rest van de wereld het huidige jaar [year] noemt, in de Maya kalender is het:
```

Comment:
Voor mij klinkt het beter om de laatste bijzin om te draaien, "is het in de Maya-kalender:"